### ISSUE TITLE
Homebridge beta 1.3.0-beta.67: characteristic was supplied illegal value: null!

### ISSUE THREAD

**Issue by dzeleski** [dgreif/ring#574]
# Bug Report

### Describe the Bug
Using the latest beta the ring plugin throws warnings around illegal values that homekit will reject.

### To Reproduce
Install beta 67 and restart homebridge

### Expected behavior
These errors should not occur.

### Screenshots/Logs
```
[18/02/2021, 10:50:27] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.
[18/02/2021, 10:50:27] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.
[18/02/2021, 10:50:27] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.
[18/02/2021, 10:50:27] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.
[18/02/2021, 10:50:27] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.
[18/02/2021, 10:50:27] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.
[18/02/2021, 10:50:27] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.
[18/02/2021, 10:50:27] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.
[18/02/2021, 10:50:27] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.
```

### Environment

 - OS: Raspbian
 - Node.js: v14.15.5
 - NPM: v6.14.11 
 - homebridge-ring: v9.15.1
 - homebridge: v1.3.0-beta.67

**Comment by dgreif** [comment]
I'm unable to recreate this issue.  When do you see those logs?  On homebridge startup, or when you try to start a live stream?

**Comment by dzeleski** [comment]
Yeah it was immediately after the upgrade and after the first restart. Im now unable to get this to occur again however. I have turned debug mode on if I do see it occur again ill post the logs.

**Comment by dgreif** [comment]
Sounds good.  Keep an eye on it and let me know if anything pops up.  Otherwise, I'll probably close this in a few days

**Comment by avojak** [comment]
I'm seeing the same error:

    [2/21/2021, 12:50:18 PM] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.

OS: Ubuntu 18.04.5
Node.js v14.15.4
NPM v6.14.10
homebridge v1.3.0
homebridge-ring v9.15.3

**Comment by TharuMCD** [comment]
Dear Team,

I am also getting this error message and cameras no longer showing images in Apple Home Kit App.  Multiple entries:

[22/02/2021, 15:44:36] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.

(repeated around a dozen times consecutively on the Homebridge UI logs.

OS: Microsoft Windows 10 Home
Node.js: v14.15.1
NPM: v7.5.4
Homebride: v1.3.0
Homebridge-Ring: v9.15.3

Kind regards,
Michael.

**Comment by gdotp01** [comment]
I’ve updated to 1.3.0 and am seeing this on startup too.

[22/02/2021, 19:38:14] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.
[22/02/2021, 19:38:14] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.
[22/02/2021, 19:38:14] [homebridge-ring] This plugin generated a warning from the characteristic 'Selected RTP Stream Configuration': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics See https://git.io/JtMGR for more info.

I think I get it for every ring device. It does not appear to impact its function. The one ring camera appears in HomeKit and is viewable.

**Comment by SlasherKG** [comment]
Same thing on the new latest HOOBS 3.3.3 release (which is based on homebridge 1.3.1).

**Comment by dgreif** [comment]
This is fixed by https://github.com/homebridge/HAP-NodeJS/commit/4f8c8c574058d5506c854c1d2b6dd24d129bc7d3.  Currently released in homebridge `1.3.4-beta.3`.  Closing this issue since it's not specific to the ring plugin and will be fixed in homebridge `1.3.4`
